src/dtclient/kubernetes_settings.go

```diff
@@ -67,33 +72,13 @@ type constraintViolations []struct {
 	Path              string
 }
 
-func (dtc *dynatraceClient) CreateOrUpdateKubernetesSetting(clusterLabel, kubeSystemUUID, scope string) (string, error) {
-	if kubeSystemUUID == "" {
-		return "", errors.New("no kube-system namespace UUID given")
-	}
-
-	body := []postKubernetesSettingsBody{
-		{
-			SchemaId:      "builtin:cloud.kubernetes",
-			SchemaVersion: "1.0.27",
-			Value: postKubernetesSettings{
-				Enabled:                         true,
-				Label:                           clusterLabel,
-				ClusterIdEnabled:                true,
-				ClusterId:                       kubeSystemUUID,
-				CloudApplicationPipelineEnabled: true,
-				OpenMetricsPipelineEnabled:      false,
-				EventProcessingActive:           false,
-				FilterEvents:                    false,
-				EventProcessingV2Active:         false,
-			},
-		},
-	}
-
-	if scope != "" {
-		body[0].Scope = scope
-	}
+const (
+	schemaId                                    = "builtin:cloud.kubernetes"
+	defaultSchemaVersion                        = "1.0.27"
+	hierarchicalMonitoringSettingsSchemaVersion = "3.0.0"
```

Comment:
is this now the `defaultSchemaVersion`?

Reply:
so we creating new setting with 2 schemas: "1.0.27" and  "3.0.0"
in case correct schema version > "3.0.0" but we try to use  "1.0.27" it still work and would not bring to error, so I suppose it is ok to have "1.0.27" as defaultSchemaVersion

Reply:
we use v3 as default now and only fallback to v1 later though. We can also just rename the constant from `defaultSchemaVersion` to `schemaVersionV1`